core/test/Voting.js

```diff
@@ -28,6 +28,10 @@ contract("Voting", function(accounts) {
     await voting.setInflationRate({ rawValue: inflationRate.toString() });
   };
 
+  const setNewGatPercentage = async gatPercentage => {
+    await voting.setGatPercentage({ rawValue: gatPercentage.toString() });
```

Comment:
Should we do the same < 100% check that we do in the constructor? And maybe add a test case for it?

Reply:
good idea. Will add now 👍